kvmagent/kvmagent/test/utils/my_debugger.py

```
@@ -0,0 +1,366 @@
+            
+            # Look for zstack-related packages
+            zstack_packages = [p for p in packages if "zstack" in p["name"].lower()]
+            if zstack_packages:
+                print("\nZStack-related packages:")
+                for pkg in zstack_packages:
+                    print("  " + pkg['name'] + "==" + pkg['version'])
+            else:
+                print("\nNo ZStack-related packages found")
+        else:
+            print("Failed to list packages: " + stderr)
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)